cellengine/utils/api_client/APIClient.py

```diff
@@ -119,6 +130,56 @@ def update_entity(self, experiment_id, _id, entity_type, body) -> dict:
             json=body,
         )
 
+    def _get_path(self, entity: CE) -> str:
+        fullpath = f"{self.base_url}/{entity.path}"
+        return fullpath
+
+    def create(self, entity: CE, **kwargs) -> CE:
+        """Create a local entity on CellEngine."""
+        if type(entity) is list:
+            return self.create_multiple(entity, **kwargs)  # type: ignore
```

Comment:
Low priority, but we should start using `@overload` to cut out this kind of `# type: ignore`. https://mypy.readthedocs.io/en/stable/more_types.html#function-overloading

Reply:
I spent some time working on overloading for this exact function, actually :laughing: I need to spend a bit more time with it, so maybe I'll leave it for another PR?

Reply:
Later is totally fine. For now, this method's return type should be `Union[CE, List[CE]]` to be usable though, right? (Otherwise users would get type errors I think.)